walk-in-interview/src/main/webapp/new-job.html

```diff
@@ -9,66 +9,49 @@
-        <input type="text" class="job-field" id="new-job-address" name="job-address" placeholder="Job Address" required>
-
-        <div class="job-field">
-            Requirements (for applicants):
-            <input type="checkbox" id="o-level" name="requirements-list" value="o-level">
-            <label for="o-level">O Levels</label>
-            <input type="checkbox" id="drivers-license" name="requirements-list" value="drivers-license">
-            <label for="drivers-license">Driver's License</label>
-            <!-- TODO(issue/17): render based on list in database -->
-        </div>
-
-        <div class="job-field">
-            Job Pay:
-            <select id="new-job-pay-frequency" name="job-pay-frequency" required>
-                <option selected disabled value="">Select</option>
-                <option value="HOURLY">Hourly</option>
-                <option value="WEEKLY">Weekly</option>
-                <option value="MONTHLY">Monthly</option>
-                <option value="YEARLY">Yearly</option>
-                <!-- TODO(issue/8): render dynamically -->
-            </select>
-            <input type="number" id="new-job-pay-min" name="job-pay-min" placeholder="min (sgd)" required>
-            -
-            <input type="number" id="new-job-pay-max" name="job-pay-max" placeholder="max (sgd)" required>
-        </div>
 
-        <div class="job-field">
-            Job Duration (optional):
-            <select id="new-job-duration" name="job-duration">
-                <option selected value="">Other</option>
-                <option value="ONE_WEEK">1 Week</option>
-                <option value="TWO_WEEKS">2 Weeks</option>
-                <option value="ONE_MONTH">1 Month</option>
-                <option value="SIX_MONTHS">6 Months</option>
-                <option value="ONE_YEAR">1 Year</option>
-                <!-- TODO(issue/8): render dynamically -->
-            </select>
-        </div>
-
-        <div class="job-field">
-            Post Expiry:
-            <input type="date" id="new-job-expiry" name="job-expiry" min="2020-07-01" max="2021-06-30" required>
-            <!-- TODO(issue/11): render min and max dynamically -->
+    <div class="content-container">
+        <div class="content-container">
+            <input class="job-field" id="new-job-title" />
+            <textarea class="job-field" id="new-job-description"></textarea>
+            <input class="job-field" id="new-job-address" />
+
+            <div class="job-field">
+                <div id="new-job-requirements-title"></div>
+                <input type="checkbox" id="o-level" name="requirements-list" value="o-level">
+                <label for="o-level">O Levels</label>
+                <input type="checkbox" id="drivers-license" name="requirements-list" value="drivers-license">
+                <label for="drivers-license">Driver's License</label>
+                <!-- TODO(issue/17): render based on list in database -->
```

Comment:
i actually recommend you render list based on javascript first. then your javascript fetches the list from database.